app/models/foreman_rh_cloud/ping.rb

```diff
@@ -0,0 +1,85 @@
+module ForemanRhCloud
+  class Ping
+    OK_RETURN_CODE = 'ok'.freeze
+    FAIL_RETURN_CODE = 'FAIL'.freeze
+
+    SERVICE_URLS = {
+      :advisor => "https://localhost:24443/api/insights/v1/status/live/",
+      :vulnerability => "https://localhost:24443/api/vulnerability/v1/apistatus"
+    }
+
+    class << self
+      def services
+        SERVICE_URLS.keys
+      end
+
+      def status
+        {
+          version: ForemanRhCloud::VERSION,
```

Comment:
Is `status` the unauthenticated one? If so, I think we should not return the version of the plugin. We've seen requests not to display version information in unauthenticated requests.

Reply:
Some additional context, @archanaserver is working on hiding the Foreman version from API responses due to a user report (https://github.com/theforeman/foreman/pull/10545). I considered this to be of a similar tack. So the answer is "probably" but not a priority compared to this work.

Reply:
Just having `version` seemed lonely, but I wasn't sure what else to put in `status`. I added the IoP flag :shrug: